### PR TITLE
add external writeup for backdoor/misc/qr

### PR DESCRIPTION
**Pull request by kbhat95**

### CODE REVIEW

**Comment by ppepos** [comment]
Thanks! Could you please update the CTF's base readme and move your challenge to External Write-ups only?

Will merge as soon as this is done =]

**Comment by kbhat95** [comment]
Done. :)

File: backdoor-ctf-2015/misc/qr/README.md [hunk 1]
```diff
@@ -17,4 +17,4 @@
 
 ## Other write-ups and resources
 
-* none yet
\ No newline at end of file
+* [http://kbhat95.github.io/backdoor-ctf/](http://kbhat95.github.io/backdoor-ctf/)
```
Comment (dhanvi) [inline backdoor-ctf-2015/misc/qr/README.md]: this can simply enclose the link inside <> instead of the one you used :+1:

Comment (kbhat95) [inline backdoor-ctf-2015/misc/qr/README.md]: Didn't know that; I'll keep it in mind. Thanks!

**Pull request merged**